File: mysql_examples/recipes/api/01_connect/connect.php
<?php
# connect.php - connect to the MySQL server

require_once "DB.php";

#@ _FRAG_
$dsn = "mysqli://cbuser:cbpass@localhost/cookbook";
$conn =& DB::connect ($dsn);
if (PEAR::isError ($conn))
  die ("Cannot connect to server\n");
print ("Connected\n");
#@ _FRAG_
$conn->disconnect ();
print ("Disconnected\n");
?>

File: mysql_examples/recipes/api/01_connect/connect.py
#!/usr/bin/python
# connect.py - connect to the MySQL server

import sys
import MySQLdb

try:
  conn = MySQLdb.connect (db = "cookbook",
                          host = "localhost",
                          user = "cbuser",
                          passwd = "cbpass")
  print "Connected"
except:
  print "Cannot connect to server"
  sys.exit (1)

conn.close ()
print "Disconnected"
